core/controlplane/config/templates/cloud-config-controller

```diff
@@ -1056,65 +1070,82 @@ write_files:
+  # Grants super-user permissions to the kube-admin user. Not actually needed
+  # in Kubernetes 1.6+ since users in the system:masters group are already
+  # given super-user access by default
   - path: /srv/kubernetes/rbac/cluster-role-bindings/kube-admin.yaml
     content: |
         kind: ClusterRoleBinding
         apiVersion: rbac.authorization.k8s.io/v1beta1
         metadata:
-          name: kube-admin
+          name: kube-aws:admin
+          annotations:
+            rbac.authorization.kubernetes.io/autoupdate: "false"
         subjects:
           - kind: User
             name: kube-admin
+          - kind: Group
+            name: system:masters
         roleRef:
           kind: ClusterRole
           name: cluster-admin
           apiGroup: rbac.authorization.k8s.io
 
-  - path: /srv/kubernetes/rbac/cluster-role-bindings/kube-worker.yaml
+  # Allows add-ons running with the default service account in kube-sytem to have super-user access
+  - path: /srv/kubernetes/rbac/cluster-role-bindings/system-worker.yaml
     content: |
         kind: ClusterRoleBinding
         apiVersion: rbac.authorization.k8s.io/v1beta1
         metadata:
-          name: kube-worker
+          name: kube-aws:system-worker
+          annotations:
+            rbac.authorization.kubernetes.io/autoupdate: "false"
         subjects:
-          - kind: User
-            name: kube-worker
+          - kind: ServiceAccount
+            namespace: kube-system
+            name: default
         roleRef:
           kind: ClusterRole
           name: cluster-admin
           apiGroup: rbac.authorization.k8s.io
 
-  # Allows add-ons running with the default service account in kube-sytem to have super-user access
-  - path: /srv/kubernetes/rbac/cluster-role-bindings/system-worker.yaml
+  # Associates the add-on group `kube-aws:node` to all nodes, so that extra kube-aws
+  # features like node draining work as expected
+  - path: /srv/kubernetes/rbac/cluster-role-bindings/node.yaml
     content: |
         kind: ClusterRoleBinding
         apiVersion: rbac.authorization.k8s.io/v1beta1
         metadata:
-          name: system-worker
+          name: kube-aws:node
+          annotations:
+            rbac.authorization.kubernetes.io/autoupdate: "false"
         subjects:
-          - kind: ServiceAccount
-            namespace: kube-system
-            name: default
+          - kind: User
+            name: system:node:kube-worker
+          - kind: Group
+            name: system:nodes
         roleRef:
           kind: ClusterRole
-          name: cluster-admin
+          name: kube-aws:node
           apiGroup: rbac.authorization.k8s.io
 
-  # Associates the add-on group `bootstrapped-node` to all nodes, so that extra kube-aws
-  # features like node draining work as expected
-  - path: /srv/kubernetes/rbac/cluster-role-bindings/bootstrapped-node.yaml
+  # Since the kube-proxy is run via the pod manifest mechanism, it's not
```

Comment:
> I don't think that working kube-proxy is a prerequisite for kube to be able to deploy DaemonSets

No, it's not.

> No for this PR, but can't we run them via DaemonSets then?

In order to create kube-proxy as a DaemonSet, it would have to be created in controller's `/opt/bin/install-kube-system`, which is not a problem. In fact, I was going to convert the kube-proxy to a DaemonSet in this PR, but since kube-aws now supports using different API endpoint URLs for different node pools, there's no way (I think) to make the DaemonSet-based kube-proxy pods to access different API endpoint URLS in different node pools.